### PR TITLE
Added get-started page

### PR DESCRIPTION
**Pull request by eyesoreee**
After a new user logged-in, they should be redirected to a page (Get-Started Page) to choose whether to continue as student, faculty, or admin.

### CODE REVIEW

File: src/layouts/GetStartedLayout.vue
Comment (fencim) [inline src/layouts/GetStartedLayout.vue:1]: you may want to use PlainLayout.vue

Comment (fencim) [reply src/layouts/GetStartedLayout.vue:1]: discard this file instead

File: src/pages/auth/LoginPage.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <q-page>
+  <q-page>npm
```
Comment (fencim) [inline src/pages/auth/LoginPage.vue]: typo error
```suggestion
  <q-page>
```

File: src/router/routes.ts [hunk 2]
```diff
@@ -2,11 +2,11 @@ import { RouteRecordRaw } from 'vue-router';
 import { IRouteMeta } from './route.meta';
 import LoginPage from 'src/pages/auth/LoginPage.vue';
 
-
 declare module 'vue-router' {
   interface RouteMeta extends IRouteMeta {
     description?: string;
     keywords?: string[];
+    role?: string | string[];
```
Comment (fencim) [inline src/router/routes.ts]: duplicate with requiresLogin
```suggestion
```

File: src/pages/auth/LoginPage.vue (hunk 1 above)
Comment (29-29) [inline src/pages/auth/LoginPage.vue]: typo as has been said
```suggestion
  <q-page>
```

**Review by 29-29 — CHANGES_REQUESTED** [review]
fix typos

**Review by fencim — APPROVED** [review]
good

**Review by 29-29 — APPROVED** [review]

**Review by fencim — APPROVED** [review]

**Review by 29-29 — COMMENTED** [review]
Can you add functionality to assign the role in the get started page? Cannot access the homepage at the moment.

**Pull request merged**